packages/backend/src/server.test.ts

```diff
@@ -50,6 +50,12 @@ describe('api', () => {
     cookie = response.headers.get('set-cookie') || ''
   })
 
+  afterEach(() => {
+    while (emailClient.sentMessages.length) {
+      emailClient.sentMessages.pop()
+    }
```

Comment:
Isn't there a way to delete all messages at once? 🤔 Or just initialize a new instance with empty messages `before` tests.